test/addons/rook-pool/start

```diff
@@ -12,16 +12,28 @@ import yaml
 import drenv
 from drenv import kubectl
 
+POOL_NAMES = ["replicapool", "replicapool-2"]
+
 
 def deploy(cluster):
+    scs = [
+        {"name": "rook-ceph-block", "pool": POOL_NAMES[0]},
+        {"name": "rook-ceph-block-2", "pool": POOL_NAMES[1]},
+    ]
 
     print("Creating StorageClass")
-    template = drenv.template("storage-class.yaml")
-    yaml = template.substitute(cluster=cluster)
-    kubectl.apply("--filename=-", input=yaml, context=cluster)
+    for sc in scs:
+        template = drenv.template("storage-class.yaml")
+        yaml = template.substitute(
+            cluster=cluster, name=sc.get("name"), pool=sc.get("pool")
```

Comment:
sc.get() will return None if we have a typo. It should be used when you want to support missing key, possibly adding a default. Since missing key is a bug in this code we want to fail loudly in this case, and the right way is:

    cluster=cluster, name=sc["name"], pool=sc["pool"]